src/renderer/extensions/whistle/index.tsx

```diff
@@ -386,28 +388,26 @@ export class WhistleExntension extends Extension {
                     }
                 });
             }, []);
-            return (
-                <>
-                    <Dropdown overlay={menu}>
-                        <div className="whistle-status-bar-item">
-                            {t('Proxy')}
-                            {port ? `: [HTTP ${port}/SOCKS5 ${((port as unknown) as number) + 1}]` : null}{' '}
-                            <Icon
-                                style={{ marginRight: '10px', marginLeft: '5px' }}
-                                className={info.proxyClassName}
-                                type={info.proxyIcon}
-                            />
-                            {t('System Proxy')}
-                            <Icon
-                                style={{ marginLeft: '5px' }}
-                                className={info.systemProxyClassName}
-                                type={info.systemProxyIcon}
-                            />
-                            <Icon style={{ marginLeft: '10px' }} type="menu" />
-                        </div>
-                    </Dropdown>
-                </>
-            );
+            return <>
```

Comment:
Maybe it's better to remove this React.Fragment? though it comes for convenience primarily

Reply:
removed